### PR TITLE
Mongo#2

### PR DESCRIPTION
**Pull request by yashraj20**

### CODE REVIEW

**Comment by yashraj20** [comment]
created separate connection file

**Comment by gowthamrodda** [comment]
@yashraj20 Configuration needs to be separated, for example the port number and mongo host should come from config file.

File: src/connection.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+let port = 2000;
+let url = 'mongodb://localhost/emp';
+
+let connection = mongo.connect(url,{useNewUrlParser: true,useUnifiedTopology: true}).then(() => { 
+    console.log('connected to MongoDB');
+},
+).catch(err => {
+    console.log("MongoDB connection error." + err);
+});
+
```
Comment (gowthamrodda) [inline src/connection.ts:15]: @yashraj20 Connection is opened but not closed.

**Review by gowthamrodda — CHANGES_REQUESTED** [review]

**Comment by gowthamrodda** [comment]
```
let con = async function connect(url): Connection {
if(url) {
return mongo.connect(url, {useNewUrlParser: true,useUnifiedTopology: true});
}
}
async function disconnect(con: Connection) {
 await con.close();
}
```
export {con, disconnect};

**Review by gowthamrodda — APPROVED** [review]

**Review by gowthamrodda — CHANGES_REQUESTED** [review]
Please solve conflicts

**Pull request closed without merging**